scripts/src/main/resources/scripts/command/python

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+
+function doSetup() {
+  if [ ! -d "${PYTHON_HOME}" ] || [ "${1}" != "silent" ]
+  then
+    # Get leatest release
+    if [ -z "${PYTHON_VERSION}" ]
+    then
+      doEcho "Python Home: ${PYTHON_HOME}"
+      doEcho "Getting latest release..."
+      # curl returns content of page, grep returns only version numbers, sort is sorting by version numbers and removing duplicates
+      # tail and head are returning the penultimate line, since it contains the latest stable version
+      PYTHON_VERSION=$(curl -L https://www.python.org/ftp/python 2>&1 | grep -o '3\.[0-9]*\.[0-9]*' | sort -u -V | tail -n2 | head -n1)
+	  fi
```

Comment:
indentation has 2 spaces to much ;)

Reply:
Fixed it myself. There was a TAB character in your code. Check your editor and disable TABs.

Reply:
you reintroduced the indentation issue again. Can you fix your editor settings and check for TABs?